lib/registries/addon/components/make-decision-dropdown/template.hbs

```diff
@@ -0,0 +1,55 @@
+<ResponsiveDropdown
+    @horizontalPosition='right'
+    as |dd|
+>
+    <dd.trigger>
+        <Button
+            data-test-moderation-dropdown-button
+            data-analytics-name='Display Moderation Actions'
+            @type='create'
+        >
+            {{t 'registries.makeDecisionDropdown.makeDecision'}}
+            <FaIcon @icon='caret-down' />
+        </Button>
+    </dd.trigger>
+    <dd.content @class={{local-class 'Dropdown'}}>
+        <ReviewActionsList @reviewActions={{this.reviewActions}} />
+        {{#each (get this.machineStateDecisionMap @registration.machineState) as |option|}}
+            <div>
+                {{#let (unique-id 'radio' option) as |uniqueId|}}
+                    <Input
+                        type='radio'
+                        @id={{uniqueId}}
+                        @value={{option}}
+                        onclick={{action this.updateDecisionTrigger option}}
+                        checked={{if (eq this.decisionTrigger option) true false}}
+                    />
```

Comment:
Since you're using built-in `<Input>`, `@type, @id, @value, @checked` must be passed in as arguments not attrs.
```suggestion
                    <Input
                        @type='radio'
                        @id={{uniqueId}}
                        @value={{option}}
                        @checked={{if (eq this.decisionTrigger option) true false}}
                        {{on 'click' (fn this.updateDecisionTrigger option)}}
                    />
```